### ISSUE TITLE
new Equinix Metal API endpoint not working from the provider?

### ISSUE THREAD

**Issue by t0mk** [packethost/terraform-provider-packet#269]
I have issues running acceptance tests for PR #268 

I run this 07.10.2020 10:13 UTC.

It might something to do with migration to api.equinix.com? Can someone verify if it's like that 

Here I try to create a project from one of the acceptance test. I supply valid API token.
```
---[ REQUEST ]---------------------------------------
POST /projects HTTP/1.1
Host: api.equinix.com
User-Agent: packngo/0.4.0
Connection: close
Content-Length: 48
Accept: application/json
Content-Type: application/json
X-Auth-Token: ...
X-Consumer-Token: aZ9GmqHTPtxevvFq9SK3Pi2yr9YCbRzduCSXF2SNem5sjB91mDq7Th3ZwTtRqMWZ
Accept-Encoding: gzip

{
 "name": "tfacc-precreated_ip_block-6de0yhnrx3"
}

-----------------------------------------------------
2020/10/07 13:02:31 [DEBUG] Packet API Response Details:
---[ RESPONSE ]--------------------------------------
HTTP/1.1 404 Not Found
Connection: close
Content-Length: 164
Content-Security-Policy: frame-ancestors 'none'
Content-Type: application/json
Date: Wed, 07 Oct 2020 10:02:32 GMT
Strict-Transport-Security: max-age=86400
X-Content-Type-Options: nosniff
X-Frame-Options: DENY
X-Xss-Protection: 1

{
 "fault": {
  "faultstring": "Unable to identify proxy for host: default and url: \/projects",
  "detail": {
   "errorcode": "messaging.adaptors.http.flow.ApplicationNotFound"
  }
 }
}
```

**Comment by thebsdbox** [comment]
Ah ok.. So..

`curl -H 'X-Auth-Token: ...' https://api.equinix.com/metal/v1/projects `-> is **fine**
`curl -H 'X-Auth-Token: ...' https://api.packet.com/projects `-> is **fine**
`curl -H 'X-Auth-Token: ...' https://api.equinix.com/projects `-> is **broken**

**Comment by t0mk** [comment]
This is caused by the new API endpoint api.equinix.com. 

I have just checked with provider release 3.0.1 (which is using the old packet endpoint), and it works OK.

**Comment by displague** [comment]
Thanks for checking in on this @t0mk. I've merged your packngo fix (we'll need a release tag on that) and then we can get this project updated.

**Comment by displague** [comment]
These changes were released with v3.1.0 which updated packngo to 0.5.0
